build.gradle

```diff
@@ -190,7 +190,7 @@ project ('spring-kafka') {
 		compile "org.springframework:spring-messaging:$springVersion"
 		compile "org.springframework:spring-tx:$springVersion"
 		compile ("org.springframework.retry:spring-retry:$springRetryVersion") {
-			exclude group: 'org.springframework'
+			exclude group: 'org.springframework', module: 'spring-core'
```

Comment:
Just to clarify a bit: we don't need to exclude any other Spring dependencies since only this one is not `optional` or `test` in that project: https://github.com/spring-projects/spring-retry/blob/master/pom.xml#L244